src/Aquifer.Public.API/Endpoints/ParentResources/List/Response.cs

```diff
@@ -0,0 +1,23 @@
+using System.Text.Json.Serialization;
+using Aquifer.Common.Utilities;
+using Aquifer.Data.Entities;
+
+namespace Aquifer.Public.API.Endpoints.ParentResources.List;
+
+public sealed class Response
+{
+    public required int Id { get; init; }
+    public required string ShortName { get; init; }
+    public required string DisplayName { get; init; }
+    public required string Code { get; init; }
+    public required int ResourceCountForLanguage { get; init; }
+
+    [JsonIgnore]
+    public string? LicenseInfoValue { get; init; }
+
+    public object? LicenseInfo => LicenseInfoValue != null
+        ? JsonUtilities.DefaultDeserialize(LicenseInfoValue)
+        : null;
```

Comment:
I bet we're doing this exact thing all over the place. Makes me wonder if this should be its own object that gets inherited when needed. Though there are probably instances where we do something similar with two different properties, and I guess you can't inherit multiple classes in C#, but might still be worth looking into (outside the scope of this ticket).

Reply:
@jwinston-bn, I'm pretty happy with this solution which removed this complexity all together and should also improve performance because instead of deserializing and serializing it keeps the JSON string as is (but still validates it before returning): https://github.com/BiblioNexusStudio/aquifer-server/pull/480/files#diff-b2898cfdfb4c4a326edb830c8f3523312ccfb2a8a18da0432384f980b3e7a6abR37